app/models/candidate_interface/satisfaction_survey_form.rb

```diff
@@ -0,0 +1,26 @@
+module CandidateInterface
+  class SatisfactionSurveyForm
+    include ActiveModel::Model
+
+    attr_accessor :question, :response
```

Comment:
Minor suggestion: maybe `answer` is a better name than `response`, which has a lot of overloaded meaning in full-stack applications.

Reply:
Good shout!